EventFilter/CTPPSRawToDigi/interface/CTPPSPixelDataFormatter.h

```diff
@@ -53,31 +55,36 @@ class CTPPSPixelDataFormatter {
   typedef std::map<int, FEDRawData> RawData;
   typedef std::vector<CTPPSPixelDigi> DetDigis;
 
+  typedef std::vector<CTPPSPixelDataError> DetErrors;
+  typedef std::map<uint32_t, DetErrors> Errors;
+
   typedef uint32_t Word32;
   typedef uint64_t Word64;
 
   CTPPSPixelDataFormatter(std::map<CTPPSPixelFramePosition, CTPPSPixelROCInfo> const &mapping);
 
+  void setErrorStatus(bool ErrorStatus);
```

Comment:
start local variables and data members with lower case letter. Capitalization is reserved for class names.

Reply:
done